populus/utils/accounts.py

```diff
@@ -12,7 +13,7 @@ def is_account_locked(web3, account):
     if isinstance(web3.providers[0], EthereumTesterProvider):
         return not any((is_same_address(account, a) for a in web3.eth.accounts[:10]))
     try:
-        web3.eth.sign(account, 'simple-test-data')
+        web3.eth.sign(account, to_bytes(text='simple-test-data'))
```

Comment:
`to_bytes` is not needed here.
Just do `w3.eth.sign(w3.eth.accounts[1], text='simple-test-data')`